Assets/Scripts/Player/PlayerSizeManager.cs

```diff
@@ -17,10 +19,19 @@ private void UpdatePlayerScale()
             var scaleIncrease = playerScore / 100;
 
             transform.localScale = Vector3.one * scaleIncrease;
+
+            playerCloth.enabled = false;
+            playerCloth.enabled = true;
```

Comment:
I assume this has good reason, but pretty strange to disable and enable. Commenting in case.

Reply:
This was that really weird fix that i was talking about

Reply:
I also couldnt test it yet because i need to merge to see, might have to couritine it